tests/nlu/base/test_server.py

```diff
@@ -15,16 +17,22 @@
 
 
 @pytest.fixture(scope="module")
-def app(tmpdir_factory):
-    """Use IResource interface of Klein to mock Rasa HTTP server.
-
-    :param component_builder:
-    :return:
-    """
+def app(tmpdir_factory, trained_nlu_model):
+    """Use IResource interface of Klein to mock Rasa HTTP server"""
 
     _, nlu_log_file = tempfile.mkstemp(suffix="_rasa_nlu_logs.json")
 
-    router = DataRouter(tmpdir_factory.mktemp("projects").strpath)
+    temp_path = tmpdir_factory.mktemp("projects")
+    try:
+        shutil.copytree(trained_nlu_model,
+                        os.path.join(temp_path.strpath,
+                                     "keywordproject/keywordmodel"))
```

Comment:
Can you declare constants, e.g. `KEYWORD_PROJECT_PATH` and `KEYWORD_MODEL_NAME`, and reuse it here and in the evaluation tests?

Reply:
Can't use `KEYWORD_PROJECT_PATH` as this depends on the tempdir which is created per test case.  I will write `KEYWORD_PROJECT_NAME` and `KEYWORD_MODEL_NAME` in `/tests/conftest.py` though